ajax/libs/coverflow/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "coverflow",
+  "filename": "coverflow.min.js",
+  "description": "A jQuery UI powered coverflow component that takes advantage of CSS3 features",
+  "homepage": "https://coverflowjs.github.io/coverflow/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/coverflowjs/coverflow.git"
+  },
+  "keywords": [
+    "coverflow",
+    "cover",
+    "album",
+    "flow",
+    "slider",
+    "slide"
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/coverflowjs/coverflow.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "coverflow.!(txt|md|json)"
```

Comment:
The better pattern would be `**/!(*.txt|*.md|*.json)`. `coverflow.!(txt|md|json)` is limited to `coverflow.`

Reply:
Done